.travis.yml

```diff
@@ -1,8 +1,33 @@
 sudo: false
 language: python
 python:
-    - "3.5"
-    - "3.6"
-    - "3.7"
+- '3.5'
+- '3.6'
+- '3.7'
 install: pip install tox-travis
 script: tox
+before_deploy:
+- python setup.py sdist
+- python setup.py bdist_wheel
+deploy:
+- provider: releases
+  skip_cleanup: true
+  file: dist/*
+  file_glob: true
+  api_key:
+    secure: 'QYPnx6ABhKhhCCTNEsDCIQ9PrjOXO8ZydrswljVQEjgVDNMSx4FBB586sYeBa8K37b6QOYvGSKM9dZGtKj5+ip1FfgyxICYQWw6LCRT0XdqBB2lXbK9mAErGZbHmx8EQP/xcZ9d+Ta0QpmY1vFbpGWrK+xFCiqKgPHROLQxE2C5bK106bacms4WeRnW/CFaD1qLNtZOYhzehfd2t+tDRN03Mx7aBMsaZLwYLDg4VuvngRsXCtWF7RBwKObcQ6js9HZ+km/R5PE5QSy8FlWy7UKnu/qZ40Ulg7OERx8Q/Kp6ETtAwkW1Fb8BuzbStM/3bPaoQiuBJ2/TBs3mttnUhSGfEJ8ZmlJR7xEpqZ89Il884p+kORAnHwe4JReKvEbSyl/IyG67kOTnfBCoc0KW0qmoLrsqBRo8i35K4i4xQu8H2c0F8g/KXMd6dzKqIjc3flPlAieFl0RVbXNdBe5sD6QkPRXzFGHV23nBsSJkUlVcMcI00xd8x4FlD6cihp1UDsW//LbQArvAXBxw2uJLFZFa5WS2PMU/iENZ8GluJyD/Q277fMN1Jha+fab1G0D8ZQZvj1emZim13zOQ/cREWf9DFjkxBYPpDK5BTQtaUtaHyIhZQS8r/yFJJeDaBr6drNzHB6PYO6ZcufeIqcx7QNVCpdOhRFVYyuqvRZBAOcok='
+  on:
+    repo: strateos/transcriptic
+    tags: true
+    python: '3.6'
+    condition: '$TRAVIS_TAG =~ ^v[0-9]+\.[0-9]+\.[0-9]+$'
+- provider: pypi
```

Comment:
Do we need `skip_existing=true`, as have multiple python environments specified?
https://docs.travis-ci.com/user/deployment/pypi/#upload-artifacts-only-once

Reply:
yup! resolving this